docs/api-docs/store-management/shipping/shipping-provider-api.mdx

```diff
@@ -496,6 +496,8 @@ When you uninstall an app with an associated shipping carrier, you also automati
   The response payload will display shipping quotes from lowest to highest price.
 </Callout>
 
+Shipping provider integrations should plan to handle incomplete postcodes when handling shipping quotes. Incomplete postcodes can occur when stores use [digital wallets](https://support.bigcommerce.com/s/article/Online-Payment-Methods?language=en_US#digital-wallet) as a payment method. Digital wallets truncate postcodes for shipping addresses in the UK and Canada as they only provide the first three characters to shipping provider integrations.
```

Comment:
Is it worth putting this somewhere more prominently or make it easier to spot like in an info/warning style panel?

I would also add that postcodes are truncated only at time of shipping quotation, and are subsequently available/complete once the order is placed (knowing that some shipping providers also read the Order data after order placement to pull details/update costs).

Reply:
Thanks @vborinsk for the feedback. I created [PR 2074](https://github.com/bigcommerce/dev-docs/pull/2074) for this if you can approve